hazelcast/src/main/java/com/hazelcast/config/security/RealmConfig.java

```diff
@@ -138,6 +139,15 @@ public RealmConfig setKerberosIdentityConfig(KerberosIdentityConfig identityConf
         return this;
     }
 
+    public AccessControlServiceConfig getAccessControlServiceConfig() {
+        return accessControlServiceConfig;
+    }
+
+    public RealmConfig setAccessControlServiceConfig(AccessControlServiceConfig accessControlServiceConfig) {
```

Comment:
Can you add some javadoc about purpose of setting an `AccessControlServiceConfig` here? It could be helpful to clarify difference between `RealmConfig#authenticationConfig` and another authentication implementation in `AccessControlService#authenticate`.